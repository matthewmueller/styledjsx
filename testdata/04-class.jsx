export default class {
  render() {
    return (
      <div>
        <p>test</p>
        <style scoped>{`
          p {
            color: red;
          }
        `}</style>
      </div>
    )
  }
}
