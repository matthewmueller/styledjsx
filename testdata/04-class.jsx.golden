import Style from "styled-jsx";
export default class {
  render() {
    return (
      <div class="jsx-4suyyD">
        <p class="jsx-4suyyD">test</p>
        <Style scoped>{`p.jsx-4suyyD{color:red}`}</Style>
      </div>
    )
  }
}
